src/controlCenter/widgets/volume/sinkInputRow.vala

```diff
@@ -20,7 +24,11 @@ namespace SwayNotificationCenter.Widgets {
 
             container = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
 
-            container.add (icon);
+            if (show_per_app_label) {
+                container.add (label);
+            } else {
+                container.add (icon);
+            }
```

Comment:
What's your opinion on showing both icon and label? Would that be too much?

Reply:
Personally I think it would get too cluttered.. But why not offer both :)  I added the option to enable/disable the label/icon separately :)